index.js

```diff
@@ -15,11 +16,19 @@ var nameMap = {
 };
 
 module.exports = function (release) {
-	var version = /\d+\.\d+/.exec(release || os.release());
+	var verRe = /\d+\.\d+/;
+	var version = verRe.exec(release || os.release());
 
-	if (!version) {
+	// workaround for Windows 10 on node < 3.1.0
+	if (!release && process.platform === 'win32' &&
+			semver.satisfies(process.version, '>=0.12.0')) {
+		var execSync = require('child_process').execSync;
+		version = verRe.exec(String(execSync('ver.exe', {timeout: 2000})));
```

Comment:
Can you wrap this in a `try/catch` with an empty catch handler, to silence any errors?